src/app/libp2p_helper/src/codanet.go

```diff
@@ -243,6 +250,10 @@ func (gs *CodaGatingState) InterceptAccept(addrs network.ConnMultiaddrs) (allow
 		gs.logGate()
 	}
 
+	// if allow && gs.InternalAddrFilters.AddrBlocked(remoteAddr) {
```

Comment:
Delete?

Reply:
I'll remove this comment and open an issue instead.